<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>graph</class>
 <widget class="QDialog" name="graph">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QSlider" name="horizontalScrollBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>620</y>
     <width>1061</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>670</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>0</y>
     <width>67</width>
     <height>203</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QCheckBox" name="c_analog_0">
      <property name="text">
       <string>Analog 0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="c_analog_1">
      <property name="text">
       <string>Analog 1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="c_analog_2">
      <property name="text">
       <string>Analog 2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="c_analog_3">
      <property name="text">
       <string>Analog 3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="c_analog_4">
      <property name="text">
       <string>Analog 4</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="c_analog_5">
      <property name="text">
       <string>Analog 5</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="c_analog_6">
      <property name="text">
       <string>Analog 6</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="c_analog_7">
      <property name="text">
       <string>Analog 7</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>210</y>
     <width>63</width>
     <height>180</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QCheckBox" name="c_digital_0">
      <property name="text">
       <string>Digital 0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="c_digital_1">
      <property name="text">
       <string>Digital 1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="c_digital_2">
      <property name="text">
       <string>Digital 2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="c_digital_3">
      <property name="text">
       <string>Digital 3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="c_digital_4">
      <property name="text">
       <string>Digital 4</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="c_digital_5">
      <property name="text">
       <string>Digital 5</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="c_digital_6">
      <property name="text">
       <string>Digital 6</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="c_digital_7">
      <property name="text">
       <string>Digital 7</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QCustomPlot" name="customPlot" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1071</width>
     <height>611</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>670</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
